nodejs/dist/uws.js

```diff
@@ -94,14 +94,27 @@ class Socket {
      *
      * @param {String|Buffer} message The message to send
      * @param {Object} options Send options
+     * @param {Function} cb optional callback
      * @public
      */
-    send(message, options) {
+    send(message, options, cb) {
         /* ignore sends on closed sockets */
-        if (!this.nativeSocket) return;
+        if(typeof options==='function'){
+            cb = options;
+            options = {};
+        }
+        if (!this.nativeSocket){ 
+            if(cb){
+                cb('Socket closed');
+            }
+            return;
+        }
 
         const binary = options && options.binary || false;
         this.server.nativeServer.send(this.nativeSocket, message, binary);
+        if(cb){
+            cb(null);
```

Comment:
True but I'm uncertain of where to put it. Unless we can provide a callback to the native send function that calls cb when it's sent.